sdks/python/container/run_validatescontainer.sh

```diff
@@ -70,22 +71,24 @@ docker -v
 gcloud -v
 
 CONTAINER=us.gcr.io/$PROJECT/$USER/$IMAGE_NAME
+TAG=$(date +%Y%m%d-%H%M%S%N)
 PREBUILD_SDK_CONTAINER_REGISTRY_PATH=us.gcr.io/$PROJECT/$USER/prebuild_python${PY_VERSION//.}_sdk
 echo "Using container $CONTAINER"
 
-# TODO(https://github.com/apache/beam/issues/27674): change this branch once jenkins is deprecated.
-  if [[ "$USER" == "jenkins" ]]; then
-    # Verify docker image has been built.
-    docker images | grep "apache/$IMAGE_NAME" | grep "$SDK_VERSION"
+if [[ "$ARCH" != "ARM" ]]; then
+  # Verify docker image has been built.
+  docker images | grep "apache/$IMAGE_NAME" | grep "$SDK_VERSION"
 
-    TAG=$(date +%Y%m%d-%H%M%S%N)
+  # Tag the docker container.
+  docker tag "apache/$IMAGE_NAME:$SDK_VERSION" "$CONTAINER:$TAG"
 
-    # Tag the docker container.
-    docker tag "apache/$IMAGE_NAME:$SDK_VERSION" "$CONTAINER:$TAG"
-
-    # Push the container
-    gcloud docker -- push $CONTAINER:$TAG
-  fi
+  # Push the container
+  gcloud docker -- push $CONTAINER:$TAG
+else
+  # Reset the test suite tag to run ARM pipelines.
```

Comment:
add a comment that ARM suite runs on GH actions only, so we don't push containers here.

Reply:
Also added a comment on how to test ARM Python SDK containers.